happi/backends/mongo_db.py

```diff
@@ -21,35 +21,55 @@ class MongoBackend(_Backend):
 
     Parameters
     ----------
-    host : str, optional
+    host : str
         Hostname for MongoDB.
-    user : str, optional
+    user : str
         Username for MongoDB instance.
-    pw : str, optional
+    pw : str
         Password for given username.
```

Comment:
There must be some better way to handle auth here but I'm not sure I have any concrete suggestions